src/game.c

```diff
@@ -309,7 +311,6 @@ void game_update()
 	// Always perform autosave check, even when paused
 	scenario_autosave_check();
 
-	network_update();
```

Comment:
I noticed there are 3 call sites to `network_update`, I removed two of them, leaving only the one in `game_logic_update` and it worked just fine in my testing, even when connecting to public servers, but I fear this could introduce unwanted changes to how we handle multiplayer. Can you look into the other call to `network_update`?